File: send.py
import pika

def sendToUser(msg):
    # establishes a connection with RabbitMQ server
    connection = pika.BlockingConnection(pika.ConnectionParameters('http://localhost:5672'))
    channel = connection.channel()

    # add 'helo' to the queue
    channel.queue_declare(queue='website')

    # sending the message to the queue_declare

    channel.basic_publish(  exchange='',
                            routing_key = 'hello',
                            body=msg)
    print (" [x] Sent " + message)

    sendToUser("sent from python")
    connection.close()

File: send.js
#!/usr/bin/env node

function sendToRabbit(msg){

  var amqp = require('amqplib/callback_api');

  amqp.connect('amqp://localhost', function(error0, connection) {
      if (error0) {
          throw error0;
      }
      connection.createChannel(function(error1, channel) {
          if (error1) {
              throw error1;
          }

          var queue = 'hello';
          //var msg = 'Hello World!';

          channel.assertQueue(queue, {
              durable: false
          });
          channel.sendToQueue(queue, Buffer.from(msg));

          console.log(" [x] Sent %s", msg);
      });
      setTimeout(function() {
          connection.close();
          process.exit(0);
      }, 500);
  });
}

//searchParameter = document.getElementById("searchParameter").value;
//searchTerm      = document.getElementById("searchTerm").value;
//msg = searchParameter + "|" + searchTerm;
sendToRabbit(["bob","toot", "scream", "Danielese"]);

//These need to be sent to receive.py
